app/views/helpers/calcBreakdownHelper.scala.html

```diff
@@ -100,6 +100,17 @@
         </tr>
         }
 
+        @if(breakdown.giftAid.taxableAmount > 0) {
+        <tr id ="gift-aid-section" class="no-border-bottom">
```

Comment:
This will cause there to be no border between this section and the total section, there needs to be logic in the previous sections (personal allowance, savings allowance, additional allowance) to handle if it needs to show the border or not.